src/appengine/libs/auth.py

```diff
@@ -59,6 +59,10 @@ def get_current_user():
   if environment.is_local_development():
     return User('user@localhost')
 
+  loas_user = environment.get_value('LOAS_PEER_USERNAME')
+  if loas_user:
+    return User(loas_user + '@google.com')
```

Comment:
Should this always be `@google.com`? I suspect that we should move it to the config (or maybe there is some way to obtain the domain dynamically), but LGTM anyway as it shouldn't cause any issues at this point!

Reply:
I believe this is specific to Google, so it does not make sense to make this configurable.

Reply:
Ack, I though LOAS is yet another SSO protocol like SAML and others :)